packages/dashboard/src/components/TfChainConnector.vue

```diff
@@ -224,8 +246,7 @@ enum SSHState {
   },
 })
 export default class TfChainConnector extends Vue {
-  /* devnet bridge for now */
-  public bridge = "GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG";
+  public bridge = (window as any).configs.APP_BRIDGE_TFT_ADDRESS;
```

Comment:
can we use the value [here](https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/dashboard/src/portal/config.ts#L5) in the config file?

Reply:
done.